AppAuth-JS/built/query_string_utils.d.ts

```diff
@@ -0,0 +1,14 @@
+import { LocationLike, StringMap } from './types';
```

Comment:
This is generated source file. All the files in built/ folder are generated JS file for the Google's lib fork